geode-core/src/distributedTest/java/org/apache/geode/internal/cache/PartitionedRegionClearDUnitTest.java

```diff
@@ -171,38 +183,172 @@ private void verifyCacheListenerTriggerCount(MemberVM serverVM) {
+    assertThat(dataStore3.invoke(getWriterClears)).isEqualTo(0);
+    assertThat(accessor.invoke(getWriterClears)).isEqualTo(1);
+
+    accessor.invoke(() -> {
+      Region region = getRegion(false);
+      region.destroyRegion();
+    });
+    assertThat(dataStore1.invoke(getWriterDestroys)).isEqualTo(0);
+    assertThat(dataStore2.invoke(getWriterDestroys)).isEqualTo(0);
+    assertThat(dataStore3.invoke(getWriterDestroys)).isEqualTo(0);
+    assertThat(accessor.invoke(getWriterDestroys)).isEqualTo(1);
+  }
+
+  @Test
+  public void normalClearFromAccessorWithoutWriterButWithWriterOnDataStore() {
+    configureServers(true, false);
+    client1.invoke(this::initClientCache);
+    client2.invoke(this::initClientCache);
+
+    accessor.invoke(() -> feed(false));
+    verifyServerRegionSize(NUM_ENTRIES);
+    accessor.invoke(() -> getRegion(false).clear());
+    verifyServerRegionSize(0);
+    // verifyCacheListenerTriggerCount(accessor);
+    assertThat(dataStore1.invoke(getWriterClears)).isEqualTo(0);
+    assertThat(dataStore2.invoke(getWriterClears)).isEqualTo(0);
+    assertThat(dataStore3.invoke(getWriterClears)).isEqualTo(1);
+    assertThat(accessor.invoke(getWriterClears)).isEqualTo(0);
+
+    accessor.invoke(() -> {
+      Region region = getRegion(false);
+      region.destroyRegion();
+    });
+    assertThat(dataStore1.invoke(getWriterDestroys)).isEqualTo(0);
+    assertThat(dataStore2.invoke(getWriterDestroys)).isEqualTo(0);
+    assertThat(dataStore3.invoke(getWriterDestroys)).isEqualTo(1);
+    assertThat(accessor.invoke(getWriterDestroys)).isEqualTo(0);
   }
 
   @Test
   public void normalClearFromClient() {
+    configureServers(true, false);
+    client1.invoke(this::initClientCache);
+    client2.invoke(this::initClientCache);
+
     client1.invoke(() -> feed(true));
     verifyClientRegionSize(NUM_ENTRIES);
     verifyServerRegionSize(NUM_ENTRIES);
 
     client1.invoke(() -> getRegion(true).clear());
     verifyServerRegionSize(0);
     verifyClientRegionSize(0);
-    verifyCacheListenerTriggerCount(null);
+    // verifyCacheListenerTriggerCount(null);
+    assertThat(dataStore1.invoke(getWriterClears)).isEqualTo(0);
+    assertThat(dataStore2.invoke(getWriterClears)).isEqualTo(0);
+    assertThat(dataStore3.invoke(getWriterClears)).isEqualTo(1);
+    assertThat(accessor.invoke(getWriterClears)).isEqualTo(0);
+
+    client1.invoke(() -> {
+      Region region = getRegion(true);
+      region.destroyRegion();
+    });
+    assertThat(dataStore1.invoke(getWriterDestroys)).isEqualTo(0);
+    assertThat(dataStore2.invoke(getWriterDestroys)).isEqualTo(0);
+    assertThat(dataStore3.invoke(getWriterDestroys)).isEqualTo(1);
+    assertThat(accessor.invoke(getWriterDestroys)).isEqualTo(0);
   }
 
   private static class CountingCacheListener extends CacheListenerAdapter {
     private final AtomicInteger clears = new AtomicInteger();
+    private final AtomicInteger destroyes = new AtomicInteger();
```

Comment:
Typo here, this should be `destroys`.

Reply:
Does this PR requires CacheListener testing.